src/syntax/parser.rs

```diff
@@ -376,7 +377,7 @@ impl<TokType: Tokenizer + Iterator<Item=Tokens>> Parser<TokType> {
 
         let expr = Expr::FnDecl(fn_name, args, return_type, definition);
 
-        return Some(ExprWrapper::default(expr));
+        Some(ExprWrapper::default(expr))
```

Comment:
Good point on being more greppable. But still, I personally haven't had any problem with it. Think of it this way: No semicolon at the end of a block? Returns.